library/general/src/modules/Popup.rb

```diff
@@ -1325,7 +1325,7 @@ def Error(message)
           Ops.greater_than(Builtins.size(lines), @too_many_lines)
         anyMessageInternalRich(
           Label.ErrorMsg,
-          message,
+          String.EscapeTags(message),
```

Comment:
I'd prefer native Ruby [ERB::Util#html_escape](https://ruby-doc.org/stdlib-2.1.2/libdoc/erb/rdoc/ERB/Util.html#method-c-html_escape) here instead of the YaST own implementation.